obsreport/obsreport_exporter.go

```diff
@@ -64,8 +64,11 @@ type ExporterSettings struct {
 	ExporterCreateSettings component.ExporterCreateSettings
 }
 
-// NewExporter creates a new Exporter.
-func NewExporter(cfg ExporterSettings) *Exporter {
+// Deprecated: [v0.64.0] use MustNewExporter.
+var NewExporter = MustNewExporter
+
+// MustNewExporter creates a new Exporter.
+func MustNewExporter(cfg ExporterSettings) *Exporter {
```

Comment:
I would expect that `NewExporter` will log the error, and `MustNewExporter` will panic. Is that not acceptable? (asking for opinions).

Reply:
Yes, I think that's acceptable, will update this pr